source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRContentStackView.mm

```diff
@@ -0,0 +1,111 @@
+//
+//  ACRContentStackView
+//  ACRContentStackView.mm
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#include "ACRContentStackView.h"
+
+@implementation ACRContentStackView
+
+-(instancetype)init 
+{
+    self = [super init];
+
+    if(self) 
+    {
+        _stackView = [[UIStackView alloc] init];
+        [self config];
+    } 
+
+    return self;
+}
+
+-(instancetype)initContentViewWithFrame:(CGRect) frame
```

Comment:
agreed

---
In reply to: [128642973](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128642973) [](ancestors = 128642973,128333362,128110303)